### PR TITLE
[WIP] syslog docs

### PR DESCRIPTION
**Pull request by murex971**

### CODE REVIEW

File: docs/artefacts/codecs.md [hunk 1]
```diff
@@ -97,3 +97,37 @@ For **gauge** there is also the field `action` which might be `add` if the value
...
+  "protocol": "RFC5424",
+  "protocol_version": 1,
+  "structured_data": {
+              "exampleSDID@32473" :
+              [
+                {"iut": "3"},
+                {"eventSource": "Application"},
+                {"eventID": "1011"}
+              ]
+            }
```
Comment (Licenser) [inline docs/artefacts/codecs.md:130]: this misses a `,` after the `}`

File: docs/artefacts/codecs.md [hunk 2]
```diff
@@ -97,3 +97,37 @@ For **gauge** there is also the field `action` which might be `add` if the value
...
+  "timestamp": 1_616_099_400_123_000_000_u64
```
Comment (Licenser) [inline docs/artefacts/codecs.md:131]: while this is really nice in rust sadly json doesn't support it :( the `_` and `_u64` won't work here

File: docs/artefacts/codecs.md [hunk 3]
```diff
@@ -97,3 +97,37 @@ For **gauge** there is also the field `action` which might be `add` if the value
 ### yaml
 
 En- and decodes [YAML](https://yaml.org).
+
+### syslog
+
+En- and decodes syslog messages (both, the standard IETF format and the old BSD format).
+A typical syslog message as follows:
```
Comment (mfelsche) [inline docs/artefacts/codecs.md:104]: Please show the formats for both standards, and if possible, link the RFCS.

It is also worth a mention that an invalid syslog message, will still be parsed, but the whole string will be put into the `msg` field.

**Review by Licenser — APPROVED** [review]
Woooh :D looks good to me!

**Review by mfelsche — APPROVED** [review]

**Pull request merged**